orc8r/tools/fab/dev_utils.py

```diff
@@ -118,12 +117,14 @@ def get_next_available_gateway_id(
     Returns:
         Next available gateway ID in the form gwN
     """
-    # gateways is a dict mapping gw ID to full resource
+    # 'gateways' is a dict mapping gw ID to full resource
     gateways = cloud_get(
         f'networks/{network_id}/gateways',
         admin_cert=admin_cert,
     )
-
+    # 'gateways' could also paginated gateways, so need to unwrap the mapping
+    if 'gateways' in gateways:
+        gateways = gateways['gateways']
```

Comment:
I think this ci check is saying to use `gateways = gateways.get('gateways', gateways)` lol. Up to you

For clarity, maybe let's rename the outer object to `res`. So then it would be

```
res = cloud_get(...)
gateways = res['gateways'] or res